aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -41,6 +41,9 @@ static inline Tensor integer_upcast(const Tensor& self, optional<ScalarType> dty
 using DimMask = TensorIterator::DimMask;
 
 static DimMask make_dim_mask(IntArrayRef dims, int64_t ndim) {
+  for(size_t idx =0; idx=dims.size(); idx++) {
```

Comment:
Please match the style of the rest of this file. The for loop should look like:
`for (size_t idx = 0; idx = dims.size(); idx++)`.